content/en/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,43 @@
+---
+reviewers:
+- derekwaynecarr
+title: PID Limiting
+content_type: concept
+weight: 10
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="stable" >}}
+
+PID Limiting features allow administrators are able to provide pod-to-pod PID isolation by defaulting the number of PIDs per pod. In addition, administrators can enable node-to-pod PID isolation by reserving a number of allocatable PIDs to user pods via node allocatable.
```

Comment:
```suggestion
Kubernetes allow you to limit the number of process IDs (PIDs) that a {{< glossary_tooltip term_id="Pod" text="Pod">}} can use. You can also reserve a number of allocatable PIDs for each {{< glossary_tooltip term_id="node" text="node">}} for use by the operating system and daemons (rather than by Pods).
```